src/SDKs/Billing/Management.Billing/Generated/Models/ErrorResponseException.cs

```diff
@@ -62,39 +58,5 @@ public ErrorResponseException(string message, System.Exception innerException)
             : base(message, innerException)
         {
         }
-
-#if LEGACY
```

Comment:
Was this handwritten code by any chance? Please check if this change was intended

Reply:
@dsgouda Not handwritten - I ran generated.cmd.

Reply:
I see that the newer code is generated by `generate.cmd`. I am concerned about the code that already exists and is being overwritten here.

Reply:
@dsgouda What options do I have?

Reply:
@dsgouda This is simply a matter of using a newer version of AutoRest. The current generated code appears to match that of other SDKs.